components/ImprovedSearch/Hit.tsx

```diff
@@ -94,12 +94,16 @@ export const Hit = ({ hit, hasQuery }: HitProps) => {
     <StyledLink href={{ pathname: "/assets/[id]", query: { id } }}>
       {community &&
         community.map((item) => (
-          <Typography variant="overline" key={item}>
+          <Typography variant="overline" style={{ fontSize: "0.7rem", fontWeight: "medium" }} key={item}>
```

Comment:
This is a really strange font size? Is it correct? `11.2px`

This should also be rendered as a `span` instead of `p`.  I probably forgot that when it was added